runtime/compiler/runtime/CompileService.cpp

```diff
@@ -46,5 +46,5 @@ void J9CompileDispatcher::compile(JITServer::ServerStream *stream)
          }
       } // end critical section
    // If we reached this point there was a memory allocation failure
-   stream->writeError(compilationLowPhysicalMemory);
+   stream->writeError(compilationLowPhysicalMemory, compInfo);
```

Comment:
Why is `compInfo` here?

Reply:
Oh, I was going to call `computeServerMemoryState(compInfo)` here but I realize it's not actually callable from here.
I'll just send `JITServer::ServerMemoryState::VERY_LOW` here, because if we are failing memory allocations it can be assumed that memory is already very low